### DISCUSSION TITLE
PODS - Logs

### DISCUSSION THREAD

**Discussion by adrianomaia** [MuhammedKalkan/OpenLens#117] [General]
So, first thing first, thanks for this amazing project. 
I have one doubt: at this moment openLens offers the possibility to see pods logs ? If not, it's in the radar for a next release ?

**Comment by treyhendon** [comment]
The code for viewing logs and exec'ing into Pods was recently changed by the upstream authors of Lens.  It now requires an extension to be installed in the application. In the Extensions menu, enter `@alebcay/openlens-node-pod-menu` as the Name and click the Install button.

Also referenced in [this](https://github.com/MuhammedKalkan/OpenLens/discussions/101) discussion.

You can view the discussion around the upstream changes [here](https://github.com/lensapp/lens/issues/6823).
